### PR TITLE
Issue #SB-30088 Registration flows enhancement - Sunbird + Gmail (Por…

### PR DESCRIPTION
**Pull request by sri-pusuluri**
…tal + Desktop app + Mobile app) UI fixes

### CODE REVIEW

File: src/app/client/src/app/modules/public/module/signup/components/signup/signup.component.ts [hunk 1]
```diff
@@ -76,7 +76,7 @@ export class SignupComponent implements OnInit, OnDestroy, AfterViewInit {
   }
 
   ngOnInit() {
-    this.signupStage = SignUpStage.BASIC_INFO;
```
Comment (rajeshkumaravel) [inline src/app/client/src/app/modules/public/module/signup/components/signup/signup.component.ts:79]: @sri-pusuluri 
please revert this change; on init the stage should be BASIC_INFO

**Review by rajeshkumaravel — CHANGES_REQUESTED** [review]

**Review by rajeshkumaravel — APPROVED** [review]

**Review by rajeevsathish — APPROVED** [review]
